Loop/Window Management/WindowEngine.swift

```diff
@@ -125,6 +125,7 @@ struct WindowEngine {
     static var frontmostWindow: Window? {
         guard
             let app = NSWorkspace.shared.runningApplications.first(where: { $0.isActive }),
+            !Defaults[.applicationExcludeList].contains(app.bundleIdentifier ?? ""),
```

Comment:
This will only work when the user has "Resize window under cursor" disabled.

Loop always will get the target window using `getTargetWindow()` (with an exception for window snapping), which when will retrieve this variable if the user has that setting disabled. However, since `getTargetWindow()` doesn't have an instance of `NSRunningApplication`, that might be pretty complicated. 

What I would recommend is to instead add a variable inside `Window`, called `isAppExcluded` (or similar), which checks if its own `nsRunningApplication`'s bundle ID is inside the excluded apps list :)

Then, inside LoopManager, you would be able to do: 
`guard self.targetWindow.isAppExcluded == true else { return }`

(let me know if you need more details, as my explanation might have been a little confusing!)